sebex/language/elixir/__init__.py

```diff
@@ -33,9 +33,12 @@ def test_project(cls, project: ProjectHandle) -> bool:
         return mix_file(project).exists()
 
     def analyze(self, project: ProjectHandle) -> AnalysisEntry:
+        import re
         with resources.path(__name__, 'elixir_analyzer') as elixir_analyzer:
             proc = popen([elixir_analyzer, '--mix', mix_file(project)])
-            raw = json.loads(proc.stdout)
+            analyzer_report = re.findall("<SEBEX_ELIXIR_ANALYZER_REPORT>(.*?)</SEBEX_ELIXIR_ANALYZER_REPORT>", proc.stdout, re.DOTALL)
```

Comment:
why the quotation mark?

Reply:
Actually this may be overkill but there is a small difference between `.*` and `.*?` - https://stackoverflow.com/a/3075140/9620900 

I think that for our use case both will be good so I can remove it

Reply:
Ok, I get it, it's just non-greedy. I think it's better to leave it as is, as we may want to print these tags multiple times one day.